crates/bevy_hierarchy/src/components/children.rs

```diff
@@ -162,3 +160,17 @@ impl<'a> IntoIterator for &'a Children {
         self.0.iter()
     }
 }
+
+#[cfg(test)]
+mod test {
+    use bevy_ecs::entity::VisitEntities;
+
+    use super::*;
+
+    fn assert_impls_map_entities<M: VisitEntities>() {}
```

Comment:
Why add such tests? You have a trait impl just a few lines above.

Reply:
Whoops, meant to remove this one as well.

It's more of a test that the `IntoIterator` implementation satisfies the blanket impl, which isn't super obvious just by looking at it. But like I mentioned in another comment, the `reflect(VisitEntities)` accomplishes the same goal.